agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -3737,6 +3737,9 @@ bool slave_thread::send_cmdu_to_controller(ieee1905_1::CmduMessageTx &cmdu_tx)
     }
 
     if (cmdu_tx.getMessageType() == ieee1905_1::eMessageType::VENDOR_SPECIFIC_MESSAGE) {
+        if (!backhaul_params.prplmesh_controller) {
```

Comment:
Ideally, we should never even get here... So maybe throw a warning if we do?

Reply:
NACK - I did this here instead of not sending the messages and checking if the controller is prplMesh everywhere else where we send the VS message.